models/staging/ecommerce/stg_ecommerce__products.yml

```diff
@@ -0,0 +1,16 @@
+
+version: 2
+
+models:
+  - name: stg_ecommerce__products
+    description: "Products data at staging phase"
+    columns:
+      - name: product_id
+      - name: product_category_name
+      - name: product_name_lenght
+      - name: product_description_lenght
```

Comment:
we have two typos to correct here!

Reply:
Updated in commit [e51990f](https://github.com/mtla-saurabh/ae-capstone-sshahdeo/commit/e51990f08f496ab9c2bfbf3e35ddf29c72b95d6e)